src/eventthread.cpp

```diff
@@ -119,6 +121,8 @@ void EventThread::process(RGSSThreadData &rtData)
 #endif
 
 	fullscreen = rtData.config.fullscreen;
+	showCursor = rtData.config.showCursor;
+	SDL_GetDesktopDisplayMode(0, &dm);
```

Comment:
as well as this

Reply:
Query the monitor resolution (
https://wiki.libsdl.org/SDL_GetDesktopDisplayMode